xonsh/environ.py

```diff
@@ -937,13 +937,13 @@ def _first_branch_timeout_message():
           file=sys.stderr)
 
 
-def current_branch(pad=True):
+def current_branch():
```

Comment:
@scopatz  I don't understand 
>I don't think that the API should be here

If we're talking about deprecation... the default behavior of the function changed anyway, is it still worth doing?